_inc/client/apps/style.scss

```diff
@@ -0,0 +1,218 @@
+.jp-landing__apps.dops-card {
+	padding: 0;
+
+	.dops-button {
```

Comment:
Is there a better selector to nest this within? Since both the apps and plans have the same CSS, should we have more generic classes that make it easier for them to share styles?